network/report.yml

```diff
@@ -0,0 +1,68 @@
+---
+# Re-write of: 
+# https://github.com/network-automation/toolkit/blob/master/playbooks/network_report.yml
+# https://github.com/network-automation/toolkit/blob/master/roles/build_report/tasks/main.yml
+
+- name: Collect facts
+  hosts: routers
```

Comment:
Set the hosts line to "{{ HOSTS }}" such that the user can select which networking devices to collect facts from?

Reply:
Good idea. I'll make it a multi-choice option in the survey as well, with the names pre-populated.